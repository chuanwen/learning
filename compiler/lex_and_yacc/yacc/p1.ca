print (1+2)*3

i = 1
x = 0
y = 0
while (i <= 100) {
    x += i
    y += (i % 3) * i^(-0.5)
    i++
}
print x
print y
