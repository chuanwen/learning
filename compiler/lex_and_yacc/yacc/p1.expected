9
5050
18.8504
